htdocs/core/lib/files.lib.php

```diff
@@ -2660,14 +2660,14 @@ function dol_check_secure_access_document($modulepart, $original_file, $entity,
 		}
 	}
 	// Fix modulepart for backward compatibility
-	if ($modulepart == 'users') {
+	if ($modulepart == 'facture') {
+		$modulepart = 'invoice';
```

Comment:
Too many impact for the moment.
Such standardization would be try when all
* all module->enabled will be replaced by isModEnabled
* all user->rights-> will be replaced with user->hasRight
* all conf->xxx->dir_output will be replaced with ->getElementProperties()
...
So impact of changing modulepart will be reduced enough to start to play with this...